data_store/db/entities.py

```diff
@@ -605,8 +605,8 @@ class Submission(BaseModel):
     submission_id = sqla.Column(sqla.String(), nullable=False, unique=True)
     reporting_round_id: Mapped[GUID] = sqla.orm.mapped_column(sqla.ForeignKey("reporting_round.id"), nullable=False)
 
-    submission_date = sqla.Column(sqla.DateTime(), nullable=True)
-    ingest_date = sqla.Column(sqla.DateTime(), nullable=False, default=datetime.now())
+    submission_date = sqla.Column(sqla.DateTime(), nullable=False, default=datetime.now())
```

Comment:
Be careful - this is copying the same mistake that `ingest_date` was making.

I would suggest it's broadly OK to not set a default time, but if you do, it needs to be `datetime.now` - specifically _not_ called. I think it's safer to not have this default as the mistake is easily made.